### PR TITLE
Milestone 3 Interactive Objects - Code Review

### PR DESCRIPTION
**Pull request by TheQuestCrafter**

### CODE REVIEW

File: Assets/Scripts/DetectLookedAtInteractive.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    }
+
+    private IInteractive lookedAtInteractive;
+
+    void FixedUpdate()
+    {
+        LookedAtInteractive = GetLookedAtInteractive();
+    }
+
+
```
Comment (JieningTao) [inline Assets/Scripts/DetectLookedAtInteractive.cs]: one line of extra white space Disgustang!

File: Assets/Scripts/InteractWithLookedAt.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
```
Comment (JieningTao) [inline Assets/Scripts/InteractWithLookedAt.cs:3]: should leave a line of white space between usings and summary

File: Assets/Scripts/InteractWithLookedAt.cs [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+/// <summary>
+/// Detects when the player presses the Interact button while looking at an IInteractive 
+/// and then calls that IInteractive's InteractWith method.
+/// </summary>
```
Comment (JieningTao) [inline Assets/Scripts/InteractWithLookedAt.cs:8]: need line of white space

File: Assets/Scripts/LookedAtInteractiveDisplayText.cs [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+    /// <summary>
+    /// Event handler for DetectLookedAtInteractive.LookedAtInteractiveChanged
+    /// </summary>
+    /// <param name="newLookedAtInteractive"></param>
+    private void OnLookedAtInteractiveChanged(IInteractive newLookedAtInteractive)
+    {
+        lookedAtInteractive = newLookedAtInteractive;
+        UpdateDisplayText();
+    }
```
Comment (JieningTao) [inline Assets/Scripts/LookedAtInteractiveDisplayText.cs:37]: need extra line of white space before region

**Review by JieningTao — COMMENTED** [review]
just some white space issues that's not major

File: Assets/Scripts/InteractWithLookedAt.cs (hunk 3 above)
Comment (TheQuestCrafter) [reply Assets/Scripts/InteractWithLookedAt.cs:8]: We talked about it and decided since it was a consistent style, it was fine.

**Pull request merged**